chain-core/src/fixed/div.rs

```diff
@@ -0,0 +1,114 @@
+use core::ops;
+
+use super::{hi_lo, neg_abs, Fixed, FRAC_NBITS, NBITS};
+
+impl ops::Div for Fixed {
+    type Output = Fixed;
+
+    #[inline]
+    fn div(self: Fixed, other: Fixed) -> Fixed {
+        let (ans, overflow) = div_overflow(self.0, other.0, FRAC_NBITS);
+        debug_assert!(!overflow, "overflow");
+        Fixed(ans)
+    }
+}
+
+#[inline]
+fn div_overflow(lhs: i128, rhs: i128, frac_nbits: u32) -> (i128, bool) {
```

Comment:
Isn't `frac_nbits` a constant in our code?

Reply:
Yes, but I think still good to keep the helper functions generic?

Reply:
Then, take `nbits` also as a parameter?

Reply:
> Then, take `nbits` also as a parameter?

These codes are just adapted from [fixed](https://gitlab.com/tspiteri/fixed) with minimal changes, so I kept the overall structures.

Reply:
Ok.

Reply:
I just made the changes, since it's only related to this single function, so it's small.